File: 3340-check-balanced-string/3340-check-balanced-string.java
class Solution {
    public boolean isBalanced(String num) {
        int se = 0;
        int so = 0;
        int i = 0;
        for(int k : num.toCharArray()){
            
        if(i % 2 == 0)
        se += k - 48;
        else
        so += k - 48;
        i++;
        }
        if(se == so)
        return true;
        return false;
        
    }
}

File: 3340-check-balanced-string/3340-check-balanced-string.c
bool isBalanced(char* num) {
    int s1=0,s2=0,i=0;
    while(num[i]!=0)
    {
        if(i==0||i%2==0)
        s1=s1+num[i]-48;
        else
        s2=s2+num[i]-48;
        i++;
    }
    if(s1==s2)
    return true;
    return false;
}